tests/test_utils.py

```diff
@@ -1845,6 +1845,22 @@ def test_replace_file_extension(index, extension, pattern, expected_index):
                 names=['idx', 'date'],
             ),
         ),
+        (
+            pd.MultiIndex.from_arrays(
+                [
+                    ['f1', 'f2'],
+                    [0, int(1e9)],
+                    [pd.NaT, pd.NaT],
+                ],
+                names=['file', 'start', 'end'],
+            ),
+            {
+                'file': 'string',
+                'start': 'timedelta64[ns]',
+                'end': 'timedelta64[ns]',
+            },
+            audformat.segmented_index(['f1', 'f2'], [0, 1]),
```

Comment:
Would be nice to have a test for `datetime64[ns]` as well as we know from `_load_csv()` that we need to use `lambda x: pd.to_datetime(x)` converters there as well.
The problem is we have no way at the moment to define the expected array then, have we?

Reply:
Isn't this already covered by this test?

```python
(
            pd.MultiIndex.from_arrays(
                [
                    ['1', '2'],
                    [None, None],
                ],
                names=['idx', 'date'],
            ),
            {
                'idx': 'int',
                'date': 'datetime64[ns]',
            },
            pd.MultiIndex.from_arrays(
                [
                    [1, 2],
                    [pd.NaT, pd.NaT],
                ],
                names=['idx', 'date'],
            ),
        ),
```

Reply:
You are right.

The array we start with has:

```
idx       int64
date    float64
dtype: object
```
and the expected array
```
idx              int64
date    datetime64[ns]
dtype: object
```